py3status/modules/uptime.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed
+
+@author Alexis "Horgix" Chotard <alexis.horgix.chotard@gmail.com>
+@license BSD
+"""
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    format = "{days} days, {hours} hours, {minutes} min"
+
+    def _uptime(self):
+        """Returns uptime in seconds or None, on Linux."""
+        with open('/proc/uptime', 'r') as f:
+            up = float(f.readline().split()[0])
+        return up
+
+    def uptime(self, i3s_output_list, i3s_config):
+        response = {}
+
+        up = self._uptime()
+        # Computing from bare seconds since timedelta only provides .days and
+        # .seconds anyway
+        days, up = divmod(up, 86400)
+        hours, up = divmod(up, 3600)
+        minutes, up = divmod(up, 60)
+        seconds = up
```

Comment:
I added a `try/except` around the file reading and made it return "None" for the full_text in case of failure.

Thanks for the suggestion !